setup.py

```diff
@@ -149,4 +149,5 @@ def get_long_description():
               },
           zip_safe=False,
           test_suite='selftests',
-          python_requires='>=2.6')
+          python_requires='>=2.6',
+          install_requires=['stevedore'])
```

Comment:
PS: I don't mean to copy whole `requirements`, but only to put there all dependencies in order to allow basic functionality without errors (we don't need PyYAML, libvirt-python, pyliblzma...)